### PR TITLE
user/magic-wormhole-rs: new package

### PR DESCRIPTION
**Pull request by tulilirockz**

### CODE REVIEW

File: user/magic-wormhole-rs/template.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                "completion",
+                shell,
+                stdout=o,
+            )
+
+
+def install(self):
+    self.install_bin(
+        f"target/{self.profile().triplet}/release/wormhole-rs",
+        name="wormhole-rs",
```
Comment (nekopsykose) [inline user/magic-wormhole-rs/template.py:33]:
```suggestion
```

already named that

File: user/magic-wormhole-rs/template.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    )
+    for shell in _supported_shells:
+        self.install_completion(
+            f"wormhole-rs.{shell}", shell, name="wormhole-rs"
```
Comment (nekopsykose) [inline user/magic-wormhole-rs/template.py:37]:
```suggestion
            f"wormhole-rs.{shell}", shell, "wormhole-rs"
```

File: user/magic-wormhole-rs/template.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+pkgname = "magic-wormhole-rs"
+pkgver = "0.7.1"
+pkgrel = 0
+build_style = "cargo"
+make_build_args = ["--workspace"]
```
Comment (nekopsykose) [inline user/magic-wormhole-rs/template.py:5]:
```suggestion
make_build_args = ["--bins"]
```

File: user/magic-wormhole-rs/template.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
+pkgname = "magic-wormhole-rs"
```
Comment (nekopsykose) [inline user/magic-wormhole-rs/template.py:1]:
```suggestion
pkgname = "magic-wormhole.rs"
```

File: user/magic-wormhole-rs/template.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+pkgver = "0.7.1"
+pkgrel = 0
+build_style = "cargo"
+make_build_args = ["--workspace"]
+hostmakedepends = ["cargo-auditable"]
+makedepends = ["rust-std"]
+pkgdesc = "Rust implementation of the Magic Wormhole protocol"
```
Comment (nekopsykose) [inline user/magic-wormhole-rs/template.py:8]:
```suggestion
pkgdesc = "Magic Wormhole CLI client"
```

File: user/magic-wormhole-rs/template.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "EUPL-1.2"
+url = "https://github.com/magic-wormhole/magic-wormhole.rs"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "c6e2acd3cccd982f449d26184d714d4cf813f51b8b75b3e36ecbb78565b3f4e8"
+_supported_shells = ["bash", "fish", "zsh"]
+# check: tests do not work because of no network connection
+options = ["!check"]
```
Comment (nekopsykose) [inline user/magic-wormhole-rs/template.py:16]:
```suggestion
# generates completions with host bin
options = ["!check", "!cross"]
```

Comment (tulilirockz) [reply user/magic-wormhole-rs/template.py:16]: i should probably should add !cross to topgrate too then! didnt know that restricted cross compilation

Comment (nekopsykose) [reply user/magic-wormhole-rs/template.py:16]: sure, yea, missed that

**Pull request merged**